x/examples/website-wrapper-app/wrapper_app_project/template/ios/App/App/AppDelegate.swift

```diff
@@ -9,12 +9,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     private var proxy: MobileproxyProxy? = nil
+    private var smartOptions: MobileproxySmartDialerOptions? = nil
 
     func application(
         _ application: UIApplication,
         didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
     ) -> Bool {
 
+        guard let decodedConfig = Data(base64Encoded: Config.smartDialer) else {
+            return false
+        }
+        let testDomains = MobileproxyNewListFromLines(Config.domainList)
+        self.smartOptions = MobileproxySmartDialerOptions(testDomains, config: String(data: decodedConfig, encoding: .utf8))
+        self.smartOptions!.logWriter = MobileproxyNewStderrLogWriter()
```

Comment:
I think using ! is bad practice. You need to use a guard instead. Perhaps we should always return a non nil object though, to avoid this

Reply:
This is because `self.smartOptions` itself is nullable (lazy initialized). I can introduce a local non-`nil` variable first and assign it to `self.smartOptions` later.